demo/src/screens/componentScreens/PickerScreen.tsx

```diff
@@ -38,6 +39,7 @@ const filters = [
 ];
 
 export default class PickerScreen extends Component {
+  picker = React.createRef();
```

Comment:
Consider exporting the relevant type and adding it here (for autocomplete and to resolve the ts error from line 290)

Reply:
I exported the type. 
Resolving the TS error on line 290 is a little more complicated than that, but at least there's now the autocomplete on the ref